src/ceph-volume/ceph_volume/devices/lvm/common.py

```diff
@@ -3,6 +3,8 @@
 from ceph_volume import terminal
 import argparse
 
+def int_str(val):
```

Comment:
I'd call it

```suggestion
def valid_osd_id(val):
```

instead. Just to improve the error message